Assets/Scripts/PlayerInventory.cs

```diff
@@ -0,0 +1,40 @@
+using UnityEngine;
+
+public class PlayerInventory : MonoBehaviour
+{
+    public LayerMask ignoreLayers;
+    public float pickUpRange;
+    public Transform hand;
+    public Vector3 inHandSize = new Vector3(0.2f, 0.2f, 0.2f);
+    public GameObject projectileHitParticles;
```

Comment:
I see that OnShoot accepted a game object before as well, but why do we really need to pass this reference along down the line?

If it's a prefab, it should be enough to just assign it in the Ammunition component directly.

Reply:
Leftover from when the Ammunition component was added to the object in the PlayerInventory (prev. PickupAmmunition) component.

Will change.